### PR TITLE
python version upgrade

### PR DESCRIPTION
**Pull request by RenuSaha**
## Description

Upgraded python version from python3.8 to python3.11

## Motivation and Context

To mitigate vulnerabilities in python3.8

## Types of changes

Modified one line in dockerfile 

## Checklist

Need to test with a FIP or FIU

### CODE REVIEW

**Comment by RenuSaha** [comment]
@gsasikumar Please review and merge

**Comment by gsasikumar** [comment]
Will it not suffice if we just rebuild this so new alpine base image would be used?

**Comment by gandha9394** [comment]
@gsasikumar , Hi I am GB. I worked with Renuka on this. We weren’t sure of the side effects it would have by upgrading the base image. From jdk version to dependencies that would change. Do you still recommend upgrading the base image instead? Plus the base image is not just alpine right? It’s some Zulu image based off of alpine. Open to hear your thoughts.

File: Dockerfile [hunk 1]
```diff
@@ -22,8 +23,12 @@ RUN /usr/lib/jvm/zulu11/bin/jlink \
     --no-header-files \
     --no-man-pages \
     --output "$JAVA_MINIMAL"
+
 # Second stage, add only our minimal "JRE" distr and our app
 FROM alpine
```
Comment (gandha9394) [inline Dockerfile]: FROM `distorless/java11`

**Review by dheerajkhardwal — APPROVED** [review]

**Pull request merged**